<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1461</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*
QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.994318, stop:0 rgba(0, 61, 101, 255), stop:1 rgba(255, 255, 255, 255))}  */



QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.652, y2:0.897727, stop:0.243781 rgba(83, 187, 255, 255), stop:1 rgba(255, 179, 249, 255))};</string>
   </property>
   <widget class="QLineEdit" name="signin_username_field">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
border: 2px solid #803CE0
}

QLineEdit:hover {
  background-color:#87cefa
};

Border-radius:10px;
font: 14pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signin_password_field">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
border: 2px solid #803CE0
}

QLineEdit:hover {
  background-color:#87cefa
};
Border-radius:10px;
font: 14pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 17pt &quot;MS Sans Serif&quot;;
font: 75 22pt &quot;MS Sans Serif&quot;;
color: rgb(70, 159, 214);</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 75 8pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>sign in to your existing account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color: rgb(4, 118, 159)
}

QPushButton:hover {
  background-color:rgb(0, 0, 0)
};

color:rgb(255, 255, 255);
font: 75 14pt &quot;MS Sans Serif&quot;;
Border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(11, 149, 245);

font: 17pt &quot;MS Sans Serif&quot;;

color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Health Management App</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>631</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Enter Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="messagefield">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="click_here_toregister">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Don't have an Account? Click here to Register.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Enter Password</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>signin_username_field</zorder>
   <zorder>signin_password_field</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>login_button</zorder>
   <zorder>label</zorder>
   <zorder>label_6</zorder>
   <zorder>messagefield</zorder>
   <zorder>click_here_toregister</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
